src/test/ui/consts/ptr_comparisons.rs

```diff
@@ -36,6 +36,10 @@ check!(!eq, 0, 1);
 check!(!ne, 0, 0);
 check!(ne, FOO as *const _, 0);
 check!(!eq, FOO as *const _, 0);
+// We want pointers to be equal to themselves, but aren't checking this yet because
+// there are some open questions (e.g. whether function pointers to the same function
+// compare equal, they don't necessarily at runtime).
+// But at least the case tested here should work.
```

Comment:
```suggestion
// The case tested here should work eventually, but does not work yet.
```
I find it very confusing when the test says "X should work" but tests the opposite.^^